model/proxy.py

```diff
@@ -86,8 +86,31 @@ def _required_attributes(self):
             'logging',
         ]
 
+    def _optional_attributes(self):
+        yield from super()._optional_attributes()
+        yield from [
+            'filter_config',
+        ]
+
     def config(self):
         return self.raw['config']
 
     def logging(self):
         return MitmLoggingConfig(self.raw['logging'])
+
+    def filter_config(self):
+        config = self.raw.get('filter_config')
+        if not config:
+            return None
+        return MitmFilterConfig(config)
+
+
+class MitmFilterConfig(ModelBase):
+    def _required_attributes(self):
+        yield from super()._required_attributes()
+        yield from [
+            'allow',
```

Comment:
whitelisted_hosts? (+blacklisted_hosts')

Reply:
also: I think this attribute should be optional (empty set / tuple if not configured)